test/parallel/test-crypto-fips.js

```diff
@@ -58,6 +59,22 @@ function testHelper(stream, args, expectedOutput, cmd, env) {
   responseHandler(child[stream], expectedOutput);
 }
 
+// --enable-fips should raise an error if OpenSSL is not FIPS enabled.
+testHelper(
+  testFipsCrypto() ? 'stdout' : 'stderr',
+  ['--enable-fips'],
+  testFipsCrypto() ? FIPS_ENABLED : FIPS_ENABLE_ERROR_STRING,
+  'require("crypto").getFips()',
```

Comment:
@danbev Isn't the point of this PR that the error message be produced _without_ having to use the crypto module?

Reply:
I can remove the last line if that make is clearer?

Reply:
> I can remove the last line if that make is clearer?

👍. I'd expect the test to demonstrate the error is thrown in the case FIPS is not available without explicitly using the crypto module, based on my understanding of what this PR is attempting to solve.